### ISSUE TITLE
Installation fails because of python-multipart-0.0.5 dependency

### ISSUE THREAD

**Issue by nahakiole** [zauberzeug/nicegui#504]
```
Collecting python-multipart==0.0.5
  Using cached python-multipart-0.0.5.tar.gz (32 kB)
  Preparing metadata (setup.py) ... error
  error: subprocess-exited-with-error
  
  × python setup.py egg_info did not run successfully.
  │ exit code: 1
  ╰─> [1 lines of output]
      ERROR: Can not execute `setup.py` since setuptools is not available in the build environment.
      [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
error: metadata-generation-failed

× Encountered error while generating package metadata.
╰─> See above for output.

note: This is an issue with the package mentioned above, not pip.
```

This doesn't happen when using python-multipart-0.0.6 

c.txt
```
python-multipart==0.0.6
```

```
$ pip install -c c.txt nicegui 
Collecting nicegui
  Using cached nicegui-1.1.10-py3-none-any.whl (3.3 MB)
......
Installing collected packages: pscript, websockets, vbuild, uvloop, uvicorn, python-engineio, python-dotenv, Pygments, markdown2, httptools, docutils, bidict, watchfiles, starlette, python-socketio, fastapi, fastapi-socketio, nicegui
Successfully installed Pygments-2.14.0 bidict-0.22.1 docutils-0.17.1 fastapi-0.88.0 fastapi-socketio-0.0.9 httptools-0.5.0 markdown2-2.4.8 nicegui-1.0.10 pscript-0.7.7 python-dotenv-1.0.0 python-engineio-4.3.4 python-socketio-5.7.2 starlette-0.22.0 uvicorn-0.20.0 uvloop-0.17.0 vbuild-0.8.1 watchfiles-0.18.1 websockets-10.4
```

System Report:
```
Python 3.10
Mac OS 12.6 (21G115)
MacBook Pro (16-inch, 2021)
Apple M1 Max
```

Can the dependency get updated to 0.0.6? Or is there a reason it's fixed at 0.0.5?

**Comment by rodja** [comment]
No reason, so I just upgraded. Will be part of next release. Thanks for pointing this out.

**Comment by rodja** [comment]
@nahakiole we just released 1.1.11 which includes this fix.
